python/sdk/prelude_sdk/models/codes.py

```diff
@@ -87,7 +87,7 @@ class ExitCode(Enum):
     def _missing_(cls, value):
         if value and not isinstance(value, int):
             return cls(int(value))
-        logging.warning('Unknown ExitCode: %s', str(value))
```

Comment:
nah, if you need to switch to black, do it across the whole python repo in a separate non-functional PR please